app/controllers/api/v1/doctors_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::V1::DoctorsController < ApplicationController
+  def index
+    @doctors = Doctor.all
+    render json: @doctors
+  end
+
+  def show
+    @doctor = Doctor.find(params[:id])
+    render json: @doctor
+  end
+
+  def create
+    @doctor = Doctor.new(doctor_params)
+    if @doctor.save
+      render json: 'Doctor created successfully'
+    else
+      render json: 'Doctor creation failed'
+    end
+  end
+
+  def doctor_params
+    params.require(:doctor).permit(:name, :speciality, :description, :graduation, :image)
+  end
```

Comment:
try to put this method in private, is exclusive for this class

Reply:
@cvalencia1991 , the doctor_param method has been made private.